scaletest/harness/run.go

```diff
@@ -132,3 +136,20 @@ func (r *TestRun) Cleanup(ctx context.Context) (err error) {
 	//nolint:revive // we use named returns because we mutate it in a defer
 	return
 }
+
+type syncBuffer struct {
+	buf *bytes.Buffer
+	mut *sync.Mutex
```

Comment:
nit, but the double ptr to the mutex is prob unnecessary since the outer struct is already a ptr